pkg/oam/util/helper.go

```diff
@@ -32,6 +32,11 @@ var (
 	ReconcileWaitResult = reconcile.Result{RequeueAfter: 30 * time.Second}
 )
 
+const (
+	//TraitPrefixKey is prefix of trait name
+	TraitPrefixKey = "trait"
```

Comment:
I don't seem to see this being used anywhere else, why make it public?

Reply:
It will be used in `GenTraitName`.

Reply:
GenTraitName is in the same package....